Sources/Protocols/CellItem/CollectionViewCellItem.swift

```diff
@@ -83,6 +83,18 @@ extension CollectionViewSiblingCellItem {
             objc_setAssociatedObject(self, &AssociatedKeys.sectionItem, newValue, .OBJC_ASSOCIATION_ASSIGN)
         }
     }
+    
+    public weak var cell: UICollectionViewCell? {
+        if let cell = objc_getAssociatedObject(self, &AssociatedKeys.cell) as? UICollectionViewCell {
+            return cell
+        }
+        printContextWarning("We found out that cell property for \(self) is nil")
+        return nil
+    }
+    
+    func update(with cell: UICollectionViewCell) {
+        objc_setAssociatedObject(self, &AssociatedKeys.cell, cell, .OBJC_ASSOCIATION_ASSIGN)
+    }
```

Comment:
IMO it is better to define getter and setter in `public weak var cell: UICollectionViewCell?` and use `cellItem.cell = cell` instead of `cellItem.update(with: cell)`.

Also we should nil `cell` after reuse ☝️

Reply:
I wan't make possible set cells from external code